src/main/kotlin/se/uulm/snowballr/backend/table/UserTable.kt

```diff
@@ -28,8 +54,22 @@ object UserTable : UUIDTable("user") {
     val role = enumeration<UserOuterClass.UserRole>("role")
     val status = enumeration<UserOuterClass.UserStatus>("status")
 
-    // Metadata
+    // User Settings
+    val showHotkeys = bool("show_hotkeys").default(SHOW_HOTKEYS_DEFAULT)
+    val reviewMode = bool("review_mode").default(REVIEW_MODE_DEFAULT)
+    val criteriaIds: Column<List<UUID>> = array<UUID>("criteria_ids").default(CRITERIA_IDS_DEFAULT)
 
+    val similarityThreshold = float("similarity_threshold").default(SIMILARITY_THRESHOLD_DEFAULT)
+    val decisionMatrix = binary(
+        "review_decision_matrix",
+    ).default(DECISION_MATRIX_DEFAULT)
+    val fetcherApis = array<String>("fetcher_apis").default(FETCHER_APIS_DEFAULT)
+    val snowballingType = enumeration<ProjectOuterClass.SnowballingType>(
+        "snowballing_type",
+    ).default(SNOWBALLING_TYPE_DEFAULT)
+    val reviewMaybeAllowed = bool("review_maybe_allowed").default(REVIEW_MAYBE_ALLOWED_DEFAULT)
```

Comment:
Let's set the default values on the client side, i.e., our server, instead of the DB. This makes it easier to change the defaults in the future because we won't require DB migration.
```suggestion
    // User settings
    val showHotkeys = bool("show_hotkeys").clientDefault { SHOW_HOTKEYS_DEFAULT }
    val reviewMode = bool("review_mode").clientDefault { REVIEW_MODE_DEFAULT }
    val criteriaIds: Column<List<UUID>> = array<UUID>("criteria_ids").clientDefault { CRITERIA_IDS_DEFAULT }

    // Project settings defaults
    val similarityThreshold = float("similarity_threshold").clientDefault { SIMILARITY_THRESHOLD_DEFAULT }
    val decisionMatrix = binary("review_decision_matrix").clientDefault { DECISION_MATRIX_DEFAULT }
    val fetcherApis = array<String>("fetcher_apis").clientDefault { FETCHER_APIS_DEFAULT }
    val snowballingType =
        enumeration<ProjectOuterClass.SnowballingType>("snowballing_type").clientDefault { SNOWBALLING_TYPE_DEFAULT }
    val reviewMaybeAllowed = bool("review_maybe_allowed").clientDefault { REVIEW_MAYBE_ALLOWED_DEFAULT }
```

Reply:
This is a good hint. I don't have any experience with database migration or that this solution exists. Thanks.